samples/healthcheck/pom.xml

```diff
@@ -100,24 +106,25 @@
               </build>
             </image>
             <image>
-              <alias>healthybox5</alias>
-              <name>busybox5</name>
+              <alias>healthy5</alias>
+              <name>nginx5</name>
               <build>
-                <from>busybox</from>
+                <from>nginx</from>
```

Comment:
Why have you changed base image to nginx? I think nginx image is bigger in size as compared to busybox.

Reply:
I was attempting to get sample to work.  Apparently, #1101 is the problem here.  I've reverted to busybox.